src/Simplex/Chat.hs

```diff
@@ -1128,9 +1136,26 @@ processChatCommand = \case
           filter (\ct -> isReady ct && not (contactConnIncognito ct))
             <$> withStore' (`getUserContacts` user)
         withChatLock "updateProfile" . procCmd $ do
-          forM_ contacts $ \ct ->
-            void (sendDirectContactMessage ct $ XInfo p') `catchError` (toView . CRChatError)
+          forM_ contacts $ \ct -> do
+            let Contact {profile = LocalProfile {profile = Profile {userPreferences = contactUserPreferences}}} = ct
+            let mergedPreferences = mergeChatPreferences globalUserPreferences contactUserPreferences
+            let mergedProfile = (p' :: Profile) {contactPreferences = mergedPreferences, userPreferences = Nothing}
```

Comment:
we should only be sending it if profile actually changed, it would be per-contact in this case, because in some cases the update of global default will not lead to the update of the user preferences for the particular contact (if override is the same as the new default).